studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -430,6 +441,18 @@ const ComputeInstanceSidePanel = () => {
                 <p>You need to disable PITR before downgrading to Micro Compute.</p>
               </Alert>
             )}
+
+            {hasChanges &&
+              subscription?.billing_via_partner &&
+              subscription.scheduled_plan_change?.target_plan !== undefined && (
+                <Alert_Shadcn_ variant={'warning'} className="mb-2">
+                  <IconAlertTriangle className="h-4 w-4" />
+                  <AlertDescription_Shadcn_>
+                    You have a scheduled subscription change that will be canceled if you change
```

Comment:
```suggestion
                    You have a scheduled subscription change that will be cancelled if you change
```

Reply:
Canceled is correct and we currently have both ways of writing it in the frontend - which one do we prefer?

Reply:
oh interesting hahaha - no worries no strong pref, can keep this